R/main.R

```diff
@@ -45,10 +81,12 @@ check_drjacoby_loaded <- function() {
 #' @param coupling_on whether to implement Metropolis-coupling over temperature 
 #'   rungs.
 #' @param GTI_pow values in the temperature ladder are raised to this power.
+#'   Provides a convenient way of concentrating rungs towards one end of the
+#'   temperature scale.
 #' @param beta_manual option to manually define temperature ladder. These values
 #'   are raised to the power \code{GTI_pow}, hence you should use \code{GTI_code
-#'   = 1} to fix powers exactly. If \code{NULL} then an equal spacing of length
-#'   \code{rungs} is used between 0 and 1.
+#'   = 1} if you want to fix powers exactly. If \code{NULL} then an equal

```

Comment:
Do values of `beta_manual` need to be beyween 0 and 1?
and
Does the length of beta_manual automatically define/override the # of rungs?

Reply:
Great question. For Metropolis-Coupling there is no requirement that rungs go all the way down to zero, so in the code we currently have no requirement for this. But I'm also trying to set us up for the next DrJ paper, in which I think we implement thermodynamic integration and talk about model comparison. In this case we will need temperatures to go to zero, so we might need some extra checks at that point.